<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="connect_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="connect_result">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="public_ip">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_keyspace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Nama Keyspace</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table_table">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>Nama Table</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="keyspace_table_name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tambah_keyspace_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Keyspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tambah_table_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyspace_and_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Keyspace: xxxx, Table: xxx</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>841</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filter_table_data">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_table_data">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>290</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapus_keyspace_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus Keyspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapus_table_btn">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>250</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tambah_table_data">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>290</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
